app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ProjectsController < ApplicationController
+def show 
+@projects = Project.find(params[:id])
+@contestants = @projects.contestants
+end
+
+def update
+  @projects = Projects.find(params[:id])
+  @projects.update(contestants)
```

Comment:
Try to come back to this extension later on for practice! I think you were pretty close; it probably doesn't know what `contestants` is in this spot.